css-four-column-layout/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title> css four column layout</title>
+  <link href="styles.css" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Barlow:wght@500&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <h2 class="title"> How it Works</h2>
+    </div>
+
+    <div class="row">
+      <div class="column-forth">
+        <div class="column">
```

Comment:
This `column` class seems confusing now with its parent being the actual `column` class. When given feedback, always make sure to analyze if that change creates any other issues in the periphery that need to be addressed.